.gitignore

```diff
@@ -18,3 +18,5 @@ client/node_modules
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+keys.js
```

Comment:
I think it needs to be `/client/keys.js`

See comments below. This is not being ignored.